spec/acceptance/postfix_exporter_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'spec_helper_acceptance'
+
+describe 'prometheus postfix exporter' do
+  describe 'install postfix' do
+    before do
+      install_module_from_forge('camptocamp/postfix', '>= 1.8.0 < 2.0.0')
+    end
+    it do
+      pp = 'include postfix'
+      apply_manifest(pp, catch_failures: true)
+      apply_manifest(pp, catch_changes: true)
+    end
+  end
+  context 'default version' do
+    it 'postfix_exporter works idempotently with no errors' do
+      pp = 'class { "prometheus::postfix_exporter": extra_options => "--systemd.enable --systemd.unit=\"postfix.service\" --postfix.logfile_path=\"\""}'
```

Comment:
You can use heredoc:

```ruby
      pp = <<<PUPPET
      class { 'prometheus::postfix_exporter':
        extra_options => '--systemd.enable --systemd.unit="postfix.service" --postfix.logfile_path=""',
      }
      PUPPET
```

Reply:
Maybe I should make those `extra_options` the default?

Reply:
since systemd is now pretty common I think it would make sense.

Reply:
this was done in the latest version.